Content.Server/Construction/ConstructionSystem.Initial.cs

```diff
@@ -203,16 +211,31 @@ void ShutdownContainers()
                             if (used.Contains(entity))
                                 continue;
 
+                            // Dump out any stored entities in used entity
+                            if (TryComp<ServerStorageComponent>(entity, out var storage) && storage.StoredEntities != null)
+                            {
+                                foreach (var storedEntity in storage.StoredEntities.ToList())
+                                {
+                                    _storageSystem.RemoveAndDrop(entity, storedEntity, storage);
+
+                                    var transform = Transform(storedEntity);
```

Comment:
Transform reparenting is already handled by `RemoveAndDrop`. Orientation randomization should probably be moved into `RemoveAndDrop`. Since that touches unrelated code, I'm just going to remove it from here and add the randomization separately.